core/src/main/scala/com/mesosphere/usi/models/USIStateEvent.scala

```diff
@@ -0,0 +1,24 @@
+package com.mesosphere.usi.models
+
+import com.mesosphere.usi.core.models.{PodId, PodRecord, ReservationId}
+
+/**
+  * General trait for snapshot or updated events, used to replicate USI's evolving view of the state.
+  *
+  * These events are emitted by the USI Scheduler Logic in response to evolution of specification states and Mesos
+  * events. They are emitted so that a framework can replicate the state and react to changes in a way that best suits
+  * the framework's domain.
+  */
+sealed trait USIStateEvent
```

Comment:
```suggestion
sealed trait StateEvent
```
I don't think we would have something non USI related in `com.mesosphere.usi.models`.

Reply:
✅